Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -23,12 +24,12 @@ public class ActorPhysics : ActorComponent
 	[ReadOnly, SerializeField]
 	ActorStates _currentState = ActorStates.Jumping;
 	ActorStateMethod CurrentStateMethod;
-	public Dictionary<ActorStates, ActorStateMethod> stateMethodMap = new Dictionary<ActorStates, ActorStateMethod>();
+	Dictionary<ActorStates, ActorStateMethod> _stateMethodMap = new Dictionary<ActorStates, ActorStateMethod>();
 	#endregion
 
 	#region Movement
 	[Header( "Movement" )]
-	public Vector3 inputVec = Vector3.zero;
+	public Vector3 _inputVec = Vector3.zero;
```

Comment:
`readonly` means that the value can only be set in the constructor, so it's not really useful in the context of MonoBehavoiurs since you don't define a constructor. Really that value should be private, `ActorPhysics` shouldn't be exposing data members, only methods.